packages/tasit-contracts/build/contracts/Migrations.json

```diff
@@ -0,0 +1,1498 @@
+{
```

Comment:
@marcelomorgado Is this `.gitignore` not working? I'm guessing you didn't want this file from the `build` dir here? Or did you?

Reply:
It's working, I've written it wrong ("./build/*" instead "build/*"). I think it will work after next commit.